docs/admin/reference/backup.rst

```diff
@@ -91,21 +121,42 @@ and ``vault`` as part of the installation process. Rename or delete any
 of these newly created AppVMs whose names conflict with the AppVMs you
 intend to restore from a backup.
 
-Example: If you restore only ``vault``, rename or delete the existing
+Example: If you wish to restore the ``vault`` VM, rename or delete the existing
 ``vault`` VM prior to restoring the backup. You can do so in
-**Q > Domain: vault > vault: Qube Settings** (the VM must not be running).
+**Q Apps ▸ vault ▸ Settings** (the VM must not be running).
 
-Restore Backup
-~~~~~~~~~~~~~~
+Restore Backup (SecureDrop Workstation components)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Plug in your backup medium and unlock it as during the backup. By default
 on a new system, your peripheral devices will be managed by a VM called
 ``sys-usb``.
 
-Navigate to **Q ▸ Qubes Tools ▸ Restore Backup**, and enter the
-location of the backup file. You do not need to adjust the default Restore
-options, unless you have made customizations to the backup. Enter the
+Navigate to **Q Application Menu ▸ Gear Icon ▸ Qubes Tools ▸ Restore Backup**,
+and enter the location of the backup file. You do not need to adjust the default
+Restore options, unless you have made customizations to the backup. Enter the
 decryption/verification passphrase, and proceed to restoring the available
-qubes (which should include the ``vault`` VM).
+qubes (which should include ``dom0`` and possibly ``sd-app``).
+
+We suggest restoring only those VMs, provisioning SecureDrop Workstation, and then
+restoring any customized VMs you may have had once that process is complete. This way
+SecureDrop Workstation is provisioned on a clean system and can implement the security
+measures it requires before any additional VMs are configured.
```

Comment:
Should these be in a `.. warning` or `.. note` block? (Answer might be no, just calling out to consider :thinking: )

Reply:
I think it's ok: I've got this in the `note` at the beginning of the Backup section, and there's a note below, so I'm trying to balance hints vs note fatigue.